main.py

```diff
@@ -464,6 +464,9 @@ def ui_access_wrapper(self, *args, **kwargs):
     get_redirect_route(
         r'/createhandler/started_tutorial_event/<exploration_id>',
         editor.StartedTutorialEventHandler),
+    get_redirect_route(
```

Comment:
@seanlip, will it be fine to change the route URL of the editor started_tutorial_event similar to this one? (not sure will this be fine to change the URL but now we have two tutorial so maybe having a similar route be good as this are not public API)

Reply:
I got it should be ok